packages/shared/src/types/firestore.ts

```diff
@@ -76,6 +76,10 @@ export interface OrganizationData {
    * for a given organization
    */
   existingSecrets?: string[];
+  /**
+   * Organization Location
+   */
```

Comment:
Maybe provide some more context to "Organization Location"...this might be used as doc (or when writing the docs) so it should be clear what it is.

Reply:
Could you please provide more context on organization location, i.e. the location of ice skating rings, or the address of the ice skating club, or something in the ball park.